android/src/main/java/com/fidelreactlibrary/adapters/FidelVerificationSetupAdapter.java

```diff
@@ -0,0 +1,34 @@
+package com.fidelreactlibrary.adapters;
+
+import com.fidelapi.Fidel;
+import com.fidelapi.entities.CardVerificationConfiguration;
+import com.fidelreactlibrary.adapters.abstraction.VerificationSetupAdapter;
+import com.facebook.react.bridge.ReadableMap;
+
+public class FidelVerificationSetupAdapter implements VerificationSetupAdapter {
+    @Override
+    public CardVerificationConfiguration map(ReadableMap data) {
+        String id = "";
+        String consentId = "";
+        String last4Digits = "";
+        ReadableMap optionsMap = data.getMap(FidelSetupKeys.CARD_CONFIG.jsName());
```

Comment:
💅 To be consistent, this is actually a (verification) configuration map, not an "options" map. It makes for "cleaner", more readable code.